Packs/CommonScripts/Scripts/GenerateAsBuilt/GenerateAsBuilt.py

```diff
@@ -482,8 +480,10 @@ def __init__(
             closed_incidents,
             playbook_stats,
             reports,
-            dashboards
-    ):
+            dashboards,
+            author,
+            customer
+    ):  # pragma: no cover
```

Comment:
Why do we need the pragma no cover here? remove this

Reply:
It's for the test coverage. The init functions of the classes are kind of big but contain no logic. But If it's not "best practice" i'll remove this.

Reply:
Yes, it is not the best practice to put this.
We only use it in specific cases.

Reply:
Done.